docs/source/support/faq.rst

```diff
@@ -115,10 +161,20 @@ require at least 4GB for RAM. Nodes also need 24/7 uptime and a static, public I
 
 For ``nucypher`` specific requirements, see `System Requirements and Dependencies <https://docs.nucypher.com/en/latest/guides/installation_guide.html#system-requirements-and-dependencies/>`_.
 
-Q: What is the network name for Incentivized Testnet?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Q: How do I set up a network node?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See :ref:`running-a-node`.
+
+Q: Is there a guide for Windows?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our guide is intended for Linux - we do not officially support Windows.
+
+Q: How computationally expensive is performing re-encryptions likely to be?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The network name for incentivized testnet is ``cassandra``.
+A re-encryption operation is very lightweight (in the order of ms) and is comparable to computing an ECDSA signature.
```

Comment:
@cygnusv - could use your help here. Is this statement correct? Would something else be more accurate? I couldn't find the previous discussion about this on discord.